website/content/docs/helm-chart/Values/0 - replicaCount.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: replicaCount
+id: replicaCount
+---
+
+# replicaCount
+
+`replicaCount` adjusts the number of replicas that the GoKubeDownscaler deployment will have.
+
+:::info
+The default value of `replicaCount` is:
+
+```yaml
+replicaCount: 1
+```
+
+:::
+
+Changing this to a different integer value will result in the deployment of the provided number of pods.
+
+:::warning
+This value should not be changed because multiple instances of the GoKubeDownscaler could lead to errors if they try to scale the same workloads simultaneously.
```

Comment:
```suggestion
This value should not be changed because running multiple instances of the GoKubeDownscaler on the same workloads is currently not supported.
This will lead to errors in the logs if they try to scale the same workload simultaneously.
```
Additionally I have made an issue for this topic with ideas on how to implement a way to allow for synchronous operation of multiple downscaler replicas.